dimod/reference/composites/higherordercomposites.py

```diff
@@ -471,3 +474,99 @@ def sample_poly(self, poly, **kwargs):
             return self.child.sample_poly(poly, **kwargs).aggregate().truncate(**tkw)
         else:
             return self.child.sample_poly(poly, **kwargs).truncate(**tkw)
+
+
+class PolyFixedVariableComposite(ComposedPolySampler):
+    """Composite to fix variables of a problem to provided.
+
+    Fixes variables of a binary polynomial and modifies linear and k-local terms
+    accordingly. Returned samples include the fixed variable
+
+    Args:
+       sampler (:obj:`dimod.Sampler`):
```

Comment:
this should be `:obj:`dimod.PolySampler`.

Reply:
Actually, it should be:
```
:class:`dimod.PolySampler`
```

Reply:
In this case the input it another object, not the type

Reply:
FWIW, I am having a hard time finding a reference/guide for `obj` vs `class`, except of course https://www.sphinx-doc.org/en/1.5/domains.html#python-roles. However, in dimod the convention has been to use `class` when referencing types or classes in the abstract and `obj` when referencing instantiated versions of those classes.

Reply:
The point I'm trying to make here is you should always list the **type** in parenthesis when describing an argument. For example, `count (int)` looks correct, right? Well, `int` is a type/class.

I believe the documentation is clear about obj/class distinction -- `:obj:` is used to reference an object of unspecified type, and `:class:` is used to reference a class (which represents a type).

Reply:
I went with :obj:`dimod.PolySampler` similar to other composites

Reply:
That was a good call, @hsadeghidw! Local consistency trumps global one. According to PEP-8, at least. :smiley: